libs/ui-components/src/hooks/useTablePagination.ts

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { ApiList } from '../utils/api';
+import { PAGE_SIZE } from '../constants';
+
+export const useTablePagination = () => {
+  const [currentPage, setCurrentPage] = React.useState<number>(1);
+  const [continueTokens, setContinueTokens] = React.useState<string[]>([]);
```

Comment:
storing previous tokens isn't optimal because as soon as the list starts changing a bit, they will be pointing to incorrect places.

But I understand we currently do not have a proper support from backend to go back so this is the only way.
We should however ask for supporting a back tokens.

That way you'd only look at the current list results. First item would be your back token and the last item would be your continue token (as is now).

Reply:
Not a blocker for this PR

Reply:
Yes, it's as you said. For going back, the risk is that the token for the previous page may be outdated. 
For moving forward, we update the token based on the last request for the current page, so that should work well.

I'll create a story for Backend to provide the back token as well.

Reply:
Created https://issues.redhat.com/browse/EDM-844 for backend and EDM-845 for UI.